Documentation/ApiOverview/Fluid/Introduction.rst

```diff
@@ -274,9 +274,9 @@ Further information
 To get an introduction to the basics of Fluid:
 
 * `The Fluid Syntax <https://github.com/TYPO3/Fluid/blob/main/doc/FLUID_SYNTAX.md>`__
-* `ViewHelpers - what these classes do in the Fluid language <https://github.com/TYPO3/Fluid/blob/main/doc/FLUID_VIEWHELPERS.md>`__
+* `ViewHelpers - what ViewHelper classes do in the Fluid language <https://github.com/TYPO3/Fluid/blob/main/doc/FLUID_VIEWHELPERS.md>`__
```

Comment:
I don' think we need to repeat ViewHelpers - what ViewHelpers classes

Reply:
I agree with @linawolf I think "these classes" is in context, so we don't need to repeat "ViewHelper".